### ISSUE TITLE
'rpmbuild -bb libhinawa.spec' fails

### ISSUE THREAD

**Issue by xpseudonym** [alsa-project/libhinawa#71]
Hi, I've just tried to build an rpm but with no luck.  I have the following errors:
$
...
RPM build errors:
    line 15: Possible unexpanded macro in: Requires: glib2(x86-64) >= %{glib2_version}
    bogus date in %changelog: Thu Aug 24 2020 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 2.2.0
    bogus date in %changelog: Thu Aug 18 2020 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 2.1.0
    bogus date in %changelog: Thu May 20 2020 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 2.0.0
    bogus date in %changelog: Thu May 11 2020 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.4.5
    bogus date in %changelog: Thu Jan 20 2020 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.4.4
    bogus date in %changelog: Sun Mar 21 2019 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.2.0
    bogus date in %changelog: Sun Mar 5 2019 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.1.2
    bogus date in %changelog: Sun Feb 25 2019 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.1.1
    bogus date in %changelog: Tue Jun 20 2018 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 1.0.0
    bogus date in %changelog: Fri May 7 2017 Takashi Sakamoto <o-takashi@sakamocchi.jp> - 0.8.1-1
    File not found: /home/admin/rpmbuild/BUILDROOT/libhinawa-2.2.0-1.el8.x86_64/usr/share/gtk-doc/html/hinawa/*
$

With 'File not found' seemingly fatal - nothing built.
Attached is the full output of the rpmbuild command:
[libhinawa_rpm-build_fail.txt](https://github.com/alsa-project/libhinawa/files/5213883/libhinawa_rpm-build_fail.txt)

Any fix/ work-around you can offer would be great.  Thanks for your help.
Morgan.

**Comment by takaswie** [comment]
Hi,

Would you check whether below patch works well?

```
diff --git a/libhinawa.spec b/libhinawa.spec
index 2b6166f..45334f4 100644
--- a/libhinawa.spec
+++ b/libhinawa.spec
@@ -34,7 +34,7 @@ developing applications that use %{name}.
 
 
 %build
-%meson
+%meson -Dgtk_doc=true
 %meson_build
 
```

Thanks

**Comment by takaswie** [comment]
fix released at v2.2.1. Thx.

**Comment by xpseudonym** [comment]
Thank you @takaswie for sorting this without my input...  I'm now running PureOS and don't have much use for `rpmbuild` any more.
